tests/Tests/200__monitor_and_manage/200__metrics/700_monitor.robot

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Common.robot
+Resource    ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library        JupyterLibrary
+Library        SeleniumLibrary
+Library        OpenShiftCLI
+Library        OperatingSystem
+
+
+*** Test Cases ***
+Verify that Prometheus image is a CPaaS built image with oc
```

Comment:
- I think this test case should be in tests/Tests/100__deploy/100__installation/102__post_install.robot
- Rename it to `Verify That Prometheus Image Is A CPaaS Built Image`
- Tier1 tag missing
- Remember to lint the code according to the style guide

Reply:
Okay
I got it